public/partials/commodity_edit.html

```diff
@@ -0,0 +1,57 @@
+<div ng-controller="NewCommodityCtrl as ctrl">
```

Comment:
@nick22891 why not try and reuse the template in `commodity_new.html` instead of re-writing this?